src/pages/ProductList/ProductList.js

```diff
@@ -1,7 +1,162 @@
 import React from 'react';
+import styled from 'styled-components';
+import { useState, useEffect } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import ProductItem from './ProductItem';
+import PlaceFilter from './Filter/PlaceFilter';
+import RecruitFilter from './Filter/RecruitFilter';
+import SortFilter from './Filter/SortFilter';
 
 const Productlist = () => {
-  return <div>응애 리스트</div>;
+  const [productitem, setProductitem] = useState([]);
+  const [placeFilterValue, setPlaceFilterValue] = useState('');
+  const [recruitFilterValue, setRecruitFilterValue] = useState('');
+  const [sortFilterValue, setSortFilterValue] = useState('');
+  const [filterDisabled, setFilterDisabled] = useState(true);
+  // const [filterValue, setFilterValue] = useState({
+  //   status: '',
+  //   participant: '',
+  //   sort: '',
+  // });
+
+  const location = useLocation();
+  const navigate = useNavigate();
+
+  // useEffect(() => {
+  //   fetch('/data/productitem.json')
+  //     .then(res => res.json())
+  //     .then(data => {
+  //       setProductitem(data);
+  //     });
+  // }, []);
+
+  const API_URL = `http://10.58.2.156:8000/places/place${location.search}`;
+
+  useEffect(() => {
+    fetch(`${API_URL}`)
+      .then(res => res.json())
+      .then(data => {
+        setProductitem(data.result);
+      });
+  }, [setProductitem, location.search, API_URL]);
+
+  useEffect(() => {
+    const queryString = `/places/place?status=${placeFilterValue}&participant=${recruitFilterValue}&sort=${sortFilterValue}`;
+    navigate(`${queryString}`);
+  }, [placeFilterValue, recruitFilterValue, sortFilterValue, navigate]);
+
+  const resetUrl = () => {
+    navigate(`/places/place?status=&participant=&sort=`);
+  };
+
+  return (
+    <Container>
+      <Filter>
+        <FilterTrack>
+          <FirstFilter>
+            <PlaceFilter
+              setPlaceFilterValue={setPlaceFilterValue}
+              placeFilterValue={placeFilterValue}
+              setFilterDisabled={setFilterDisabled}
+            />
+          </FirstFilter>
+          <SecondFilter>
+            <RecruitFilter
+              setRecruitFilterValue={setRecruitFilterValue}
+              recruitFilterValue={recruitFilterValue}
+              filterDisabled={!filterDisabled}
+            />
+          </SecondFilter>
+          <ThirdFilter onClick={() => resetUrl()}>필터 초기화</ThirdFilter>
```

Comment:
```suggestion
          <ThirdFilter onClick={resetUrl}>필터 초기화</ThirdFilter>
```
* 이 함수가 전달하는게 없다면, 함수 자체를 넣어주세요

Reply:
수정하였습니다!